proposals/lip-0070.md

```diff
@@ -178,7 +178,13 @@ Transactions executing this command have:
 
 ##### Parameters
 
-The `params` property of claim transactions is empty.
+```java
+claimRewardsParams = {
+    "type": "object",
+    "required": [],
+    "properties": {}
+}
+```
 
 ##### Verification
```

Comment:
The schema validation in the `verify` hook is missing.

Reply:
added